src/main/java/com/kodilla/ecommercee/domain/dto/GroupDto.java

```diff
@@ -0,0 +1,15 @@
+package com.kodilla.ecommercee.domain.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+
+@Getter
+@AllArgsConstructor
+@NoArgsConstructor
+public class GroupDto {
+    private Long id;
+    private String name;
+
```

Comment:
Hej, patrząc na diagram ERD wydaje mi się że chyba jeszcze pole productId powinno być...
I nazwę pierwszego id dobrze by było zmienić na zgodną z diagramem, czyli zamiast samego id -> groupId (tak właśnie używasz w controlerze nawet :) )

Reply:
Wcześniej miałem uwagę, że w Json nie ma zwracanego productId to w Dto też nie musi być. Nazwe id zmienie :)

Reply:
Ok :)

Reply:
Jak będzie się coś nie zgadzać to zawsze można w kolejnych krokach to dodać :)